cli/src/deploy/mod.rs

```diff
@@ -131,25 +136,25 @@ pub async fn new(agz_file: &str, cloud_alias: &str) {
   let sp = SpinnerBuilder::new(format!("Creating new app in {}", cloud_alias)).start();
   let resp = post_v1("new", body).await;
   let res = match resp {
-    Ok(app_id) => format!("Created app with id {} in {} succesfully!", app_id, cloud_alias),
+    Ok(cluster_id) => format!("Created app with id {} in {} succesfully!", cluster_id, cloud_alias),
```

Comment:
Not in this PR, but nit: `succesfully` -> `successfully`

Reply:
oh thanks, fixing!